packages/databricks-vscode/package.json

```diff
@@ -6,7 +6,7 @@
     "license": "LicenseRef-LICENSE",
     "version": "2.0.2",
     "engines": {
-        "vscode": "^1.86.0"
+        "vscode": "^1.88.0"
```

Comment:
Yesterday we've got a feedback from a customer that vscode version requirements can be confusing. When you try to install extension using usupported vscode version it doesn't really show any errors or messages - it seems like the extension is installed, but just doesn't work.

One way to minimize the confusion is to have the requirement clearly stated in docs, but another one is to relax the requirements here. Can we do that, or we actually need `1.88.0` version?

Reply:
We need this for supporting checkbox change events.